RockLib.HealthChecks/RockLib.HealthChecks.csproj

```diff
@@ -10,10 +10,10 @@
 		<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
 		<PackageReleaseNotes>A changelog is available at https://github.com/RockLib/RockLib.HealthChecks/blob/main/RockLib.HealthChecks/CHANGELOG.md.</PackageReleaseNotes>
 		<PackageTags>rocklib health checks</PackageTags>
-		<PackageVersion>3.0.0</PackageVersion>
```

Comment:
I thought you already updated and published this?  Does this branch need a rebase?